### ISSUE TITLE
Feature request: Indicate which disk(s) are mounted

### ISSUE THREAD

**Issue by mrdoornbos** [GideonZ/1541ultimate#214]
Would be a nice to have (not need to have) to be able to see what disk(s) are currently mounted.

**Comment by GideonZ** [comment]
Do you mean to which file it is linked in the virtual file system, or what the contents are of the disk?
What if the disk is mounted in "unlinked" mode? Would you still want to see the original filename?
And if the filename were to be displayed, would this need to include the whole path?

**Comment by mrdoornbos** [comment]
Yeah, I guess really whatever would be an equivalent to "There's a disk in drive 8" and something about it that helps identify what and how it's in there. I assume contents of the disk I would just do via traditional methods.

**Comment by GideonZ** [comment]
Okay.. How would you like this to be integrated in the UI?  Like a menu option "Show status"? For quick reference that may be a but cumbersome. There is no free screen real estate to show this info permanently.

**Comment by Grrrolf** [comment]
##  _**Just thinking out loud:**_

What about a status screen linked to F4? Pressing F4 will then open the status screen. F6 is already in use for other (debug?) information (at least on the U2+, U64 it's empty). This makes the information quickly accessible.

This page can then show e.g. information about the mounted disks, which type of drive it is inserted and if it's enabled. Or only show the enabled one. It can even show the clock so that one can quickly see how much it deviates ;-)

It could look like this: 
```
Drive Status information:

Drive A:      Enabled
Drive type:   1541-II
Mounted disk: this_is_file_name.d65
path:         /Usb0/directory1/directory/2directory3
              /directory4/directory5/    (when too long, let it flow to the next line)

Drive B:      Enabled
Drive type:   1541
Mounted disk: blank_disk.g64. (disk inserted using 'Insert blank disk').
path:         None
```
When any of the drives are disabled only show that the drive is disabled. However, if someone might have inserted or mounted a disk, it can show both that the drive is disabled and the information in the above example. Or do not allow mounting/inserting blank disks when the drive is disabled.
```
Drive B:      Disabled
```
Instead of only showing drive status information it can also be used to show a  few things so that one does. not need to dive into the configuration menu to find out things like:
```
Joy port swapped          yes  (u64 elite only)
Cartridge image:          None
Ram expansion unit:       enabled
REU size:                 16MB 
Map Ultimate Audio:       disabled
IEC drive and printer:    disabled
Modem – ACIA (6551) Mode: disabled
Command interface:        enabled 
```
This second part exceeds the requirements by @mrdoornbos and can be skipped since it likely unnecessarily adds valuable development time which can be spend on more important things. Starting with just the drive info + path + name of mounted disk would suffice.

**Comment by mrdoornbos** [comment]
Yeah would be helpful for sure.

**Comment by GideonZ** [comment]
Implemented in 3.10.
